src/bam/collapse_reads_to_fragments/calc_consensus.rs

```diff
@@ -79,6 +87,41 @@ impl<'a> CalcOverlappingConsensus<'a> {
     fn overlap(&self) -> usize {
         self.overlap
     }
+    fn build_consensus_strand(
+        &self,
+        consensus_strand: &mut Vec<u8>,
+        ref_base: u8,
+        base_pos: usize,
+    ) {
+        let mut strands = HashSet::new();
+        let fwd_end_pos = self.recs1()[0].len();
```

Comment:
Calling this fwd is a bis misleading. I'd rather say first and recond read or so.